app/dashboard/models.py

```diff
@@ -392,11 +399,11 @@ def status(self):
                 return 'unknown'
 
     @property
-    def value_true(self):
+    def _value_true(self):
```

Comment:
Can we move away from using `_` for public methods?

Reply:
done